adapters/index.ts

```diff
@@ -0,0 +1,14 @@
+import { AdapterRequest, AdapterResponse } from '@chainlink/types'
+import * as dnsquery from '../dns-query/src'
```

Comment:
These can also be imported from workspaces, instead of direct path, if we keep this functionality. Also could be moved to subfolders under the adapters folder.